### ISSUE TITLE
method does not override or implement a method from a supertype

### ISSUE THREAD

**Issue by piotrgorecki** [rt2zz/react-native-contacts#236]
Hi,
I have got the following error when I try to use @1.0.5 with RN 0.49.3 as well as 0.50.4.
Build @1.0.4 works correctly.

```
.../node_modules/react-native-contacts/android/src/main/java/com/rt2zz/reactnativecontacts/ReactNativeContacts.java:26: error: method does not override or implement a method from a supertype
    @Override 
    ^
1 error
:react-native-contacts:compileReleaseJavaWithJavac FAILED
```

**Comment by morenoh149** [comment]
according to https://github.com/facebook/react-native-fbsdk/pull/345 we should remove the `@Override` syntax. Anyone wanna submit a PR?

**Comment by morenoh149** [comment]
Please test now. Followed suggestions at https://github.com/facebook/react-native-fbsdk/pull/345

**Comment by itinance** [comment]
I have tested. Current latest release will break upon the `@Override`. We need to remove this.

**Comment by morenoh149** [comment]
I already removed the line and released v2.0.0 of this project. See https://github.com/rt2zz/react-native-contacts/commit/95a5f87ff5904a24457b1b15abf9da3043b322e0

**Comment by piotrgorecki** [comment]
Thank you @morenoh149, now it works correctly (v2.0.0).
